apps/admin/backend/src/store.ts

```diff
@@ -1847,6 +1851,127 @@ export class Store {
     return rows;
   }
 
+  /**
+   * Gets write-in record adjudication queue for a specific contest.
+   */
+  getWriteInAdjudicationCvrQueue({
```

Comment:
The following three queries follow the same pattern used in the existing write-in flow, though now we adjudicate on a cvr by cvr basis, rather than individual write-ins.

Relatedly, I expect to add a follow up PR or commit that deletes all the unneeded code for the existing write-in flow, depending on if that is the expected process when deprecating a screen